examples/layout/lakes/interactive/lib/main.dart

```diff
@@ -95,8 +112,8 @@ class MyApp extends StatelessWidget {
       mainAxisAlignment: MainAxisAlignment.center,
       children: [
         Icon(icon, color: color),
-        Container(
-          margin: const EdgeInsets.only(top: 8),
+        Padding(
+          padding: const EdgeInsets.only(top: 8),
```

Comment:
Padding is different from margin, but that may be just fine. Padding is on the inside of the shape, and margin is on the outside. If the container doesn't have a color or decoration, then it really doesn't matter much.